src/components/test/context.jsx

```diff
@@ -7,6 +7,7 @@ import styles from './test.css';
 
 const cx = classNames.bind(styles);
 
+const base64RegEx = /^data:image\/([a-zA-Z]*);base64,([^"]*)$/i;
```

Comment:
It's probably a non issue but this RegExp doesn't support image MIME types with dashes or dots. Also, I'd like to see a few more test cases for the most likely image types just so we're covered.